config/config.default.php

```diff
@@ -229,6 +229,9 @@
     // Users are able to sign up
     'registration_enabled'    => (bool) env('REGISTRATION_ENABLED', true),
 
+    // URL to external register page
```

Comment:
Should mention that this functionality is only available when `enable_password` is `false`. Ans naming it `external_registration_url` would imho fit better with the current naming scheme